Vagrantfile

```diff
@@ -172,9 +172,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     vb.customize ['modifyvm', :id, '--nictype1', 'virtio']
   end
 
-  # Use vagrant-vbguest plugin to make sure Guest Additions are in sync
-  config.vbguest.auto_reboot = true
-  config.vbguest.auto_update = true
+  # Disable auto-update as it causes plugin failures
+  config.vbguest.auto_update = false
```

Comment:
This is the default if we are indeed in "command mode", [again as per the documentation](https://github.com/dotless-de/vagrant-vbguest#running-as-a-command). If we are in command mode, I suggest we remove this line altogether.

Reply:
As per the documentation, the `auto_update` is for:

> auto_update (Boolean, default: true) : Whether to check the correct additions version on each start (where start is not resuming a box).

If you remove the line, it will do auto-update (the default value is true) and we want to avoid that.

Reply:
TL;DR: Sorry, I made a mistake and the original patch is good.

My bad! I misinterpreted your patch and thought you were setting `auto_reboot` to `False`. The way it is perfectly fine. Sorry about the churn.